### PR TITLE
Introduce script for processing aura bribes, rename other processor and introduce reward distr addr

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #684 

```
brownie run badger/process_bribes_graviaura claim_and_sell_for_weth

brownie run badger/process_bribes_graviaura sell_weth

brownie run badger/process_bribes_graviaura emit_tokens
```

**Note:** that to process tokens correctly, we're dependant on nonce [1062 in dev_msig](https://gnosis-safe.io/app/eth:0xB65cef03b9B89f99517643226d76e286ee999e77/transactions/0xf7e58d690dcc0366f0a9329c7bb54c38c677f0fb978bb2ca030d04b91e588f07) which sets the processor into the strat.

### CODE REVIEW

File: scripts/badger/process_bribes_graviaura.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+def claim_and_sell_for_weth():
+    bribes_dest = GreatApeSafe(PROCESSOR.address)
+    bribes_dest.take_snapshot(r.bribe_tokens_claimable.values())
+
+    claimed = SAFE.badger.claim_bribes_hidden_hands()
+
+    for addr, mantissa in claimed.items():
+        order_payload, order_uid = SAFE.badger.get_order_for_processor(
+            sell_token=SAFE.contract(addr),
+            mantissa_sell=mantissa,
```
Comment (jalbrekt85) [inline scripts/badger/process_bribes_graviaura.py:39]:
```suggestion
            mantissa_sell=int(mantissa),
```

**Review by jalbrekt85 — CHANGES_REQUESTED** [review]

File: scripts/badger/process_bribes_graviaura.py (hunk 1 above)
Comment (jalbrekt85) [reply scripts/badger/process_bribes_graviaura.py:39]: `mantissa` is `str` here

Comment (petrovska-petro) [reply scripts/badger/process_bribes_graviaura.py]: good catch!

Comment (petrovska-petro) [reply scripts/badger/process_bribes_graviaura.py]: even tho, i think will be resolved [here](https://github.com/Badger-Finance/badger-multisig/blob/main/great_ape_safe/ape_api/cow.py#L53) also along the execution of the code

**Review by jalbrekt85 — DISMISSED** [review]
lgtm

**Review by jalbrekt85 — APPROVED** [review]
ignores posting orders for aura and badger now, lgtm

**Review by sajanrajdev — APPROVED** [review]
- Introduces Great Ape Safe function to claim bribes from HH
- Introduces the required Aura bribes processing functions (NOTE: Properly avoids adding BADGER and AURA to the Cowswap quote for wETH as the Processor won't allow it and goes against the purpose. A separate PR will introduce better estimation of the amount of BADGER and AURA to buy based on the received bribes of these assets. For this round it is ok to move forward without this since Bribes were only received in the form of USDC).
- Tested with the staging Cowswap platform - Bribes were claimed properly and a order for USDC -> wETH was submitted properly.

LGTM

**Pull request merged**